x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -99,7 +99,7 @@ export const mapToCard = ({
     url: uiInternalPathUrl,
     fromIntegrations: selectedCategory,
     integration: 'integration' in item ? item.integration || '' : '',
-    name: 'name' in item ? item.name || '' : '',
+    name: 'name' in item ? item.name : item.id,
```

Comment:
UI link cards did not have a name, only an ID with `ui_link` prefix, this populates the name with what you would expect. e.g the apm card name is now 'apm' and the web crawler card 'web_crawler'